gdal/src/main/resources/reference.conf

```diff
@@ -0,0 +1,4 @@
+vlm.gdal {
```

Comment:
`raster.gdal`?

Reply:
👍 `vlm` is definitely confusing ('cause I want the **real** VLM!)